crates/cli-support/src/test_runner.rs

```diff
@@ -0,0 +1,627 @@
+    filter: Option<String>,
+
+    /// Use [Bindgen::no_modules] when generating test sources.
+    no_modules: bool,
+
+    /// Disable headless tests.
+    no_headless: bool,
+
+    /// Use [Bindgen::debug] when generating test sources.
+    debug: bool,
+
+    /// A directory to use in place of a generated temp directory.
+    temp_dir: Option<PathBuf>,
+
+    /// The timeout to use for webdriver processes.
+    driver_timeout: u64,
+
+    /// The timeout to use for browser processes.
+    browser_timeout: u64,
+
+    /// Use [Bindgen::split_linked_modules] when generating test sources.
+    split_linked_modules: bool,
+
+    /// The webdriver json config content.
+    capabilities_content: Option<String>,
+
+    /// An optional prefix
+    coverage_profraw_prefix: Option<String>,
+
+    /// The expected output file or directory for coverage profraw files.
+    coverage_profraw_out: Option<PathBuf>,
+
+    /// Skip setting Cross-Origin policies for tests.
+    no_origin_isolation: bool,
+
+    /// An optional override address to use for headless tests.
+    address: Option<String>,
+
+    /// The test mode to use when the type is not identifiable from [TestRunner::file].
+    fallback_test_mode: Option<TestMode>,
+
+    /// Only run web tests.
+    test_only_web: bool,
+
+    /// Only run node tests.
+    test_only_node: bool,
+
+    /// The path to the desired `nodejs` binary for node tests.
+    node_bin: Option<PathBuf>,
+
+    /// Augment `NODE_PATH` so things like `require("tests/my-custom.js")` work
+    /// and Rust code can import from custom JS shims. This is a bit of a hack
+    /// and should probably be removed at some point.
+    node_path: Vec<PathBuf>,
+
+    /// Arguments to pass to nodejs for node tests.
+    node_args: Vec<String>,
+}
+
+impl TestRunner {
+    /// Constructor for the [TestRunner].
+    pub fn new(file: PathBuf) -> Self {
+        Self {
+            file,
+            include_ignored: false,
+            ignored: false,
+            exact: false,
+            skip: Vec::new(),
+            list: false,
+            nocapture: false,
+            format: None,
+            filter: None,
+            no_modules: false,
+            no_headless: false,
+            debug: false,
+            temp_dir: None,
+            driver_timeout: 5,
+            browser_timeout: 20,
+            split_linked_modules: false,
+            capabilities_content: None,
+            coverage_profraw_prefix: None,
+            coverage_profraw_out: None,
+            no_origin_isolation: false,
+            address: None,
+            fallback_test_mode: None,
+            test_only_node: false,
+            test_only_web: false,
+            node_bin: None,
+            node_path: Vec::new(),
+            node_args: Vec::new(),
+        }
+    }
+
+    pub fn with_include_ignored(&mut self) -> anyhow::Result<&mut Self> {
```

Comment:
```suggestion
    pub fn include_ignored(&mut self) -> anyhow::Result<&mut Self> {
```
To align with `Bindgen`.

Reply:
Done